macros/get_user_unsubscribed_message_type_columns.sql

```diff
@@ -0,0 +1,24 @@
+{% macro get_user_unsubscribed_message_type_columns() %}
+
+{% if does_table_exist('user_unsubscribed_message_type') %}
+
+{% set columns = [
+    {"name": "_fivetran_synced", "datatype": dbt.type_timestamp()},
+    {"name": "_fivetran_id", "datatype": dbt.type_string()},
+    {"name": "message_type_id", "datatype": dbt.type_string()}
+] %}
+
+{% else %}
+
+{% set columns = [
+    {"name": "_fivetran_synced", "datatype": dbt.type_timestamp()},
+    {"name": "email", "datatype": dbt.type_string()},
+    {"name": "message_type_id", "datatype": dbt.type_string()},
+    {"name": "updated_at", "datatype": dbt.type_timestamp()}
+] %}
+
+{% endif %}
```

Comment:
See the same comments within `get_user_unsubscribed_channel_columns` for similar updates that should be applied here.

Reply:
updated!